app/_locales/en/messages.json

```
@@ -1147,6 +1132,26 @@
     "message": "from",
     "description": "Part of the sentence describing when and where snap was added"
   },
+  "flaskWelcomeUninstall": {
+    "message": "uninstall this extension immediately",
+    "description": "This request is shown on the Flask Welcome screen. It is intended for non-developers, and will be bolded."
+  },
+  "flaskWelcomeWarning1": {
+    "message": "Flask is for developers to experiment with new unstable APIs. Unless you are a developer intending to use Flask for experimentation, please $1.",
+    "description": "This is a warning shown on the Flask Welcome screen, intended to encourage non-developers not to proceed any further. $1 is the bolded message 'flaskWelcomeUninstall'"
+  },
+  "flaskWelcomeWarning2": {
+    "message": "We do not guarantee the safety or stability of this extension. The new APIs offered by Flask are not hardened against phishing attacks, meaning that any site or snap that requires Flask might be a malicious attempt to steal your assets.",
+    "description": "This explains the risks of using MetaMask Flask"
+  },
+  "flaskWelcomeWarning3": {
+    "message": "Developers, please understand that all Flask APIs are experimental. They may be changed or removed without notice, or they might stay on Flask indefinitely without ever been migrated to stable MetaMask. Do not rely upon these APIs in production.",
```

Oops, this is wrong.

```suggestion
    "message": "Developers, please understand that all Flask APIs are experimental. They may be changed or removed without notice, or they might stay on Flask indefinitely without ever being migrated to stable MetaMask. Do not rely upon these APIs in production.",
```